docs/dictionary/command/libUrlSetDriver.lcdoc

```diff
@@ -0,0 +1,35 @@
+Name: libURLSetDriver
+
+Type: command
+
+Syntax: libURLSetDriver <driverName>
+
+Summary: 
```

Comment:
I'm fairly sure summaries are required. Also would I be able to trouble you to hard wrap it to 72 columns?